cmd/dep/testdata/harness_tests/init/govendor/case1/final/Gopkg.toml

```diff
@@ -0,0 +1,9 @@
+ignored = ["github.com/sdboyer/dep-test"]
+
+[[constraint]]
+  name = "github.com/carolynvs/go-dep-test"
```

Comment:
dep doesn't support wildcards [yet](https://github.com/golang/dep/pull/709), but we can at least try to preserve at least some of the intent behind ignoring a directory.

Glide supports something similar (`excludeDir`) and what the glideImporter does is join the project root of the project being importer (it's passed to `Import`) with the relative path to create a dep ignore entry. [Here's the unit test](https://github.com/golang/dep/blob/master/cmd/dep/glide_importer_test.go#L282).